torch/_logging/_registrations.py

```diff
@@ -90,5 +90,10 @@
     "Detailed Inductor fusion decisions. More detailed than 'schedule'",
     off_by_default=True,
 )
+register_artifact(
+    "overlap",
```

Comment:
I am wondering if we should merge this `overlap` to the `distributed` logger? Most likely we only see this when we turn on distributed, and if we turn on distributed debug logger + the inductor config, we should see this, wdyt?

Reply:
synced offline, seems `TORCH_LOGS=“distributed”` not working yet, we will merge these two later

Reply:
it should be working now. ping me for how to set it up?